editor/src/components/canvas/__snapshots__/ui-jsx-canvas.spec.tsx.snap

```diff
@@ -21572,21 +22062,30 @@ exports[`UiJsxCanvas render renders fragments correctly 1`] = `
         bottom: 0;
         background-color: #ffffff;
       \\"
+      data-paths=\\"eee/fff:aaa :eee/fff\\"
       data-uid=\\"aaa fff\\"
     >
       <div
         data-label=\\"random-div\\"
         style=\\"width: 100px; height: 100px;\\"
         data-uid=\\"bbb\\"
+        data-paths=\\"eee/fff:aaa/25d80840-fbda-4984-be3b-43838cb78079\\"
       >
         Hello
         <div
           data-label=\\"some-other-div\\"
           style=\\"width: 100px; height: 100px;\\"
           data-uid=\\"ccc\\"
+          data-paths=\\"eee/fff:aaa/25d80840-fbda-4984-be3b-43838cb78079/8e84c2c0-99bd-40dc-bc63-4d8230425414\\"
```

Comment:
hmmmmmmmmm, what turns `ccc` into `25d80840-fbda-4984-be3b-43838cb78079` ? 😂 

at least this uid is truly random so this broke the test, helping me find it...